srsly/_json_api.py

```diff
@@ -61,11 +68,18 @@ def write_json(location, data, indent=2):
     """
     json_data = json_dumps(data, indent=indent)
     if location == "-":  # writing to stdout
-        print(json_data)
+        if use_gzip:
+            print(gzip.compress(json_data.encode('utf-8')))
```

Comment:
Are we going to have a problem from `print` here? gzip will output bytes. I'm not sure what the right way to send bytes to stdout is in Python 3.

Reply:
Does it make sense to want to write the data to stdout in this case anyways? 

Maybe we should just have separate helper functions like `write_json_gzip` and `read_json_gzip` that just don't support stdin/stdout. This also lets us keep things clean and avoid too many options and conditionals on the main helpers.

Reply:
> Maybe we should just have separate helper functions

I avoided this at first because I thought the function signatures were close so I thought it'd be a small change, but the implementation ended up more awkward than I expected... It's probably easiest to add these as separate functions like you suggest, so I'll do that for now. That also lets us avoid the question of what to do with print/stdout for the time being.